package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Parse and consume binary streams with a neat DSL",
   "main": "index.js",
   "engines": {
-    "node": "~0.10.0"
+    "node": ">= 0.10.0 <= 5.10.0"
```

Comment:
I don't think this version specifier is a good idea. What about `>= 0.10.0 ^5` instead?